lib/models/subscription.js

```diff
@@ -39,6 +39,7 @@ module.exports = (sequelize, DataTypes) => {
 
   const Subscription = sequelize.define('Subscription', {
     githubId: DataTypes.BIGINT,
+    githubName: DataTypes.STRING,
```

Comment:
same question as above 😬 -- Could we add a comment about what it's used for?

Reply:
Done here as well 👍